addons/l10n_ph/migrations/1.2/end-migrate_update_taxes.py

```diff
@@ -0,0 +1,34 @@
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+from odoo import api, SUPERUSER_ID
```

Comment:
Wrong order for import
[Guidelines](https://www.odoo.com/documentation/master/contributing/development/coding_guidelines.html#imports)